### ISSUE TITLE
Problem opening the "root" URI

### ISSUE THREAD

**Issue by dancvrcek** [writingink/wink#176]
Hi,

I have installed wink pretty quickly - it got own database, created the schema, ... all good stuff.

I have a problem to access the root URI - let's say I set WINK_PATH="blog".

I can get to test/blog/login but once submitted, it redirects to the authenticated part of my main Laravel app (which has authenticated content).

For a while, when I tried test/blog, nginx returned 403 error. I uninstalled wink and reinstalled. This time it forwards it to the main login page.

Also, when I run  

`php artisan route:list|grep blog`

it doesn't list the "blog" but all subpages are there.

Have you come across this before?

**Comment by dancvrcek** [comment]
The second part of my problem - redirection to authetication of the main applications has already been solved in #139 . In my case, adding a condition:
```
if (!substr($path_info, 1, strlen(config("wink.path"))) === config("wink.path")) {
    // do the existing code
}
```

in OnUserLogin() listener solved the problem.

Regarding the routing - my silly - I've put it under 'wink' middleware.

**Comment by GigaMick** [comment]
For what it's worth I'm having this exact same issue.
